.storybook/themes/commerce.scss

```
@@ -680,6 +680,35 @@
 						),
 					),
 				),
+				right-rail: (
+					display: grid,
+					gap: var(--global-spacing-6),
+				),
+				right-rail-navigation: (
+					position: sticky,
+					top: 0,
+					width: 100%,
+					z-index: 9,
+				),
+				right-rail-main: (
+					display: grid,
+					gap: var(--global-spacing-6),
+					width: 100%,
+				),
+				right-rail-rail-container: (
+					max-width: initial,
+					width: 100%,
+					margin: auto,
+					grid-template-columns: initial,
+					grid-template-rows: 1fr,
+					gap: var(--global-spacing-5),
+				),
+				right-rail-main-right-rail: (
+					grid-column: initial,
+				),
+				right-rail-main-interior-item: (
+					grid-column: initial,
```

As we are setting this property to make it more understandable on what we are doing if we set the value to `1`
```suggestion
					grid-column: 1,
```

---

@matthewroach, @nschubach  added the suggestion and thanks for the explanation guys, appreciate it.